pytest_plugins/rerun_rp/rerun_rp.py

```diff
@@ -83,25 +83,28 @@ def pytest_addoption(parser):
 
         Defect_types:
             Collect failed tests marked with defect types: {[*ReportPortal.defect_types.keys()]}
-        '''
+    '''
     parser.addoption("--only-failed", const='all', nargs='?', help=help_text)
     help_text = '''
         Collects only skipped tests in Report Portal to run only skipped tests.
 
         Usage: --only-skipped
-        '''
+    '''
     parser.addoption("--only-skipped", action='store_true', default=False, help=help_text)
     help_text = '''
-            Collects user tests from report portal.
+        Collects user tests from report portal.
 
-            Usage: --user [value]
+        Usage: --user [value]
 
-            Value: [ Report_portal_username ]
-
-            Defect_types:
-                Collect tests of specific user from Report Portal
-        '''
+        Value: [ Report_portal_username ]
+    '''
     parser.addoption("--user", nargs='?', help=help_text)
+    help_text = '''
+        Rerun Upgrade launch tests instead of Regular Tier launch from Report Portal,
+
+        Usage: --upgrade-rerun
+    '''
+    parser.addoption("--upgrades-rerun", action='store_true', default=False, help=help_text)
```

Comment:
"upgrade-tests-rerun" is more appropriate than "upgrades-rerun"

Reply:
@devendra104 As pytest and rerun plugin is used for running tests and this plugin specially plays with test collection, so I think I need not to explicitly add `test` word there.

So I am good with whats it now.